templates/cloud/plans-and-pricing.html

```diff
@@ -82,12 +82,11 @@
     </div>
 </section><!-- end row -->
 
-<section class="row strip-light">
+<section class="row strip-light" id="bootstack-fully-managed-cloud">
```

Comment:
@barrymcgee I wanted to target that row to send the direct link to someone, and couldn't. I personally think all our rows should have IDs for targeting in links, but unfortunately at the moment there's not an easy way to automate that. So I just added this one as an infinitesimal step in the right direction.